react/features/settings/components/native/SettingsView.tsx

```diff
@@ -590,17 +726,16 @@ class SettingsView extends AbstractSettingsView<Props, State> {
  * @param {Object} state - The Redux state.
  * @returns {Props}
  */
-function _mapStateToProps(state) {
+function _mapStateToProps(state: any) {
     const localParticipant = getLocalParticipant(state);
-    const localParticipantId = localParticipant?.id;
-    const avatarLabel = localParticipant && getParticipantDisplayName(state, localParticipantId);
 
     return {
-        ..._abstractMapStateToProps(state),
+        _serverURL: getDefaultURL(state),
         _serverURLChangeEnabled: isServerURLChangeEnabled(state),
-        avatarLabel,
-        localParticipantId
+        _visible: state['features/settings'].visible,
+        localParticipantId: localParticipant?.id,
```

Comment:
Nit: we prefix all props that originate in mapStateToProps with _ to tell them apart from the own props.